astropy/coordinates/solar_system.py

```diff
@@ -30,20 +30,20 @@
 DEFAULT_JPL_EPHEMERIS = 'de430'
 
 """List of kernel pairs needed to calculate positions of a given object."""
-BODY_NAME_TO_KERNEL_SPEC = OrderedDict(
-                                      (('sun', [(0, 10)]),
-                                       ('mercury', [(0, 1), (1, 199)]),
-                                       ('venus', [(0, 2), (2, 299)]),
-                                       ('earth-moon-barycenter', [(0, 3)]),
-                                       ('earth', [(0, 3), (3, 399)]),
-                                       ('moon', [(0, 3), (3, 301)]),
-                                       ('mars', [(0, 4)]),
-                                       ('jupiter', [(0, 5)]),
-                                       ('saturn', [(0, 6)]),
-                                       ('uranus', [(0, 7)]),
-                                       ('neptune', [(0, 8)]),
-                                       ('pluto', [(0, 9)]))
-                                      )
+BODY_NAME_TO_KERNEL_SPEC = OrderedDict([
+    ('sun', [(0, 10)]),
+    ('mercury', [(0, 1), (1, 199)]),
+    ('venus', [(0, 2), (2, 299)]),
+    ('earth-moon-barycenter', [(0, 3)]),
+    ('earth', [(0, 3), (3, 399)]),
+    ('moon', [(0, 3), (3, 301)]),
+    ('mars', [(0, 4)]),
+    ('jupiter', [(0, 5)]),
+    ('saturn', [(0, 6)]),
+    ('uranus', [(0, 7)]),
+    ('neptune', [(0, 8)]),
+    ('pluto', [(0, 9)])
```

Comment:
Wait a minute... I thought Pluto got demoted. :wink:

Reply:
Out of scope ;)